snap/snapcraft.yaml

```diff
@@ -17,7 +17,7 @@ apps:
     command: CodeChecker/bin/CodeChecker
     environment:
       PATH: ${SNAP}/usr/bin:${PATH}
-      PYTHONPATH: ${SNAP}/usr/lib/python3.8
+      PYTHONPATH: ${SNAP}/usr/lib/python3.9
```

Comment:
I honestly don't know if this is correct.

Reply:
This is all that can be done with snaps to specify a Python version.